src/tsc/tsconfig.release.json

```diff
@@ -1,7 +1,7 @@
 {
     "extends": "../tsconfig-base",
     "compilerOptions": {
-        "outFile": "../../lib/tsc.js",
+        "outFile": "../../built/local/tsc.release.js",

```

Comment:
...this was potentially building into lib? 😱

Reply:
Yep.